### ISSUE TITLE
how to get the score like the paper？

### ISSUE THREAD

**Issue by lancelee98** [engrodawood/deep-Phurie#1]
I use your pre-trained model to evaluate 2015 IRWIN and 2016 IRWIN，but get rmse at 22.66kt and 23.39kt?
I have done the post-processing wrote on paper, but still cann't get rmse under 10kt?
how to get low mse like that in your paper?

![post_2015_20 86_15 83_0 33_](https://user-images.githubusercontent.com/35135275/113419339-d84d0c80-93f9-11eb-86bf-d97f5c057fa2.png)

The picture above is the predicted speed of the model in 2015 IRWIN dataset.
N means the number of IR pics.

**Comment by engrodawood** [comment]
Regarding your first comment are you doing pre-processing on image? If yes what kind of pre-processing you are using. 
You need to remove images where hurricane hit land. Secondly in this datasets if you analyse the images there are some noisy images you need the remove those noisy images before generating the prediction.
Regarding postprocessing the results without postprocessing seems to be in around 10knot. I am don't remember the exact figure for 2015 hurricane but you can confirm it. In the paper mean rmse is reported for leave one hurricane out. So for some hurricane the rmse was a bit higher that other. But I am sure it was not 20knot like the one you are getting.

**Comment by lancelee98** [comment]
Thank you for your detailed answer. 
Because the scipy.misc.imresize is discard. Therefore I alter some of your code. 
But I did something wrong in the pre-precessing.  Now I find the problem and have fixed it. 


```
def get_x_y(nc_path):
    f = nc4.Dataset(nc_path, 'r')
    im = f.variables['IRWIN']
    y = f.variables['WindSpd'][0]
    img = np.array(im)
    img = np.reshape(img, (301, 301))
    # resize the image to 224 * 224 
    # img = Image.fromarray(img.astype('uint8'))# origin wrong code
    img = Image.fromarray(img)  # right code
    imnew = img.resize((224, 224))
    x = image.img_to_array(imnew)
    return x, y
```

Because I didn't find the hit-land label in the .nc file ,so i didn't exclude the hurricane that hit land.
Although I didn't exclude the hurricane that hit land, I can still get good rmse at around 10kt.  
It's indeed a good work. Thank you for your detailed answer again.

![image](https://user-images.githubusercontent.com/35135275/113471191-5d353600-948d-11eb-9ecb-45f042e42e1d.png)
